packages/scanner/ml_kit/lib/src/scanner_ml_kit.dart

```diff
@@ -107,7 +138,7 @@ class _SmoothBarcodeScannerMLKitState extends State<SmoothBarcodeScannerMLKit>
                 MobileScannerException error,
                 Widget? child,
               ) =>
-                  EMPTY_WIDGET,
+                  const SizedBox.shrink(),
```

Comment:
You may as well create another shared `EMPTY_WIDGET` in the new `Scanner` files, as it's used several times.